ddp-workspace/projects/ddp-cgc/src/assets/i18n/en.json

```diff
@@ -0,0 +1,400 @@
+      },
+      {
+        "Letter": "C",
+        "Title": "We will work with you to obtain your genetic test results.",
+        "Description": "<p>Our study team will work with you via email to obtain your genetic test results from the appropriate company/lab.</p>"
+      },
+      {
+        "Letter": "D",
+        "Title": "We will gather your medical record data.",
+        "Description": "<p>We will request your medical records from your physicians and hospitals. We will extract the relevant information aggregate these results. You also have access to these records. If you want a copy of the records we have requested, email cancergenomecollective@broadinstitute.org and we will send your medical records to you.</p>"
+      },
+      {
+        "Letter": "E",
+        "Title": "We will aggregate the genetic and medical record data.",
+        "Description": "<p>We will then take the important information from your medical record, along with your genetic information and combine it to look for patterns. As we discover new connections, we will share this with you and the rest of world via presentations, publications, and blog posts.</p>"
+      },
+      {
+        "Letter": "F",
+        "Title": "We will share the data with other researchers.",
+        "Description": "<p>In the future, we will share this data with the greater research community.</p>"
+      },
+      {
+        "Letter": "G",
+        "Title": "Become an active researcher!",
+        "Description": "<p>We are excited to learn with you! Throughout the project, we will provide all patients with regular updates about the status of the project and information about new studies that we are conducting. We will also share any discoveries from these research projects with you.</p><p>We also encourage you to submit your own research ideas by sending an email to cancergenomecollective@broadinstitute.org. We will do our best to conquer any research ideas put forward by our patients!</p>"
+      }
+    ],
+    "Info": "Please contact us at <b>cancergenomecollective@broadinstitute.org</b> or <b>651-229-5806</b> with any questions, comments, or suggestions."
+  },
+  "ErrorPage": {
+    "Title": "Something went wrong",
+    "Text": "We’re sorry, it looks like something unexpected happened. Please try refreshing your page or logging in again."
+  },
+  "StayInformed": {
+    "Title": "Thank you for your interest!",
+    "Paragraph1": "Unfortunately, at this time, all of the statements on the previous page must be true to participate.",
+    "Paragraph2": "Please join our mailing list to be informed about the project and any updates on future participation.",
+    "Form": {
+      "Email": {
+        "Label": "Email Account*",
+        "Error": "Please enter a valid Email Account"
+      },
+      "Submit": "I want to stay informed"
+    }
+  },
+  "PasswordPage": {
+    "Form": {
+      "Password": {
+        "Label": "Please enter your password",
```

Comment:
This wording is a bit confusing. Password page asks for the site password and not some user's password. How about `Please enter the site password`?

Reply:
Yes, it is better, I fixed it.